console/console-init/ui/src/modules/iot-project-detail/containers/IoTProjectDetailHeaderContainer/IoTProjectDetailHeaderContainer.tsx

```diff
@@ -86,21 +85,17 @@ export const IoTProjectDetailHeaderContainer: React.FC<IIoTProjectDetailHeaderCo
     });
   };
 
-  // TODO: HANDLE AFTER MOCK IS READY
-  const handleChangeEnabled = () => {
+  const handleChangeEnabled = (checked: boolean) => {
```

Comment:
handleChangeStatus would  be more better name.

Reply:
Done!